### PR TITLE
Refactor loading hierarchy for nft selector to improve perceived speed

### PR DESCRIPTION
**Pull request by kaitoo1**
### Summary of Changes

Currently, the query to load the Nft Selector puts the whole screen in a skeleton loading state, blocking the whole UI.
This makes the Post flow feel slow especially on mobile networks.
Since the bulk of the query is for populating the nft grid, I moved it closer to where the data is used so that other static UI elements can be rendered immediately.

The other fragment in the query is used to get the experience flag to show the Creator Announcement bottom sheet. I refactored this too to separate it into its own query and wrap it in its own suspense. This way it doesn't block the rest of the screen from loading.

Also noticed the header text was missing so fixed that

### Demo or Before/After Pics

Before:
https://github.com/gallery-so/gallery/assets/80802871/bf7005fb-61f7-4a0a-821e-558eb41a1d0d

After:
https://github.com/gallery-so/gallery/assets/80802871/0a11b0da-fa1b-4f84-b1cb-b8f8e13cdb58

Bottom sheet still works:
https://github.com/gallery-so/gallery/assets/80802871/16c27dc5-5dcd-4be2-8f10-bde0cd84f7b0

### Edge Cases
simulated slow networks 

### Testing Steps
Go to the post composer or pfp selector

### Checklist

Please make sure to review and check all of the following:

- [ ] I've tested the changes and all tests pass.
- [ ] (if web) I've tested the changes on various desktop screen sizes to ensure responsiveness.
- [ ] (if mobile) I've tested the changes on both light and dark modes.

### CODE REVIEW

File: apps/mobile/src/screens/NftSelectorScreen/NftSelectorPickerScreen.tsx [hunk 1]
```diff
@@ -243,20 +221,61 @@ function InnerNftSelectorPickerScreen() {
...
+        <Suspense>
+          <CreatorBottomSheetWrapper isViewingCreatedFilter={ownershipTypeFilter === 'Created'} />
+        </Suspense>
       </View>
-      <CreatorSupportAnnouncementBottomSheet ref={announcementBottomSheetRef} />
     </>
   );
 }
 
+function CreatorBottomSheetWrapper({
```
Comment (jakzaizzat) [inline apps/mobile/src/screens/NftSelectorScreen/NftSelectorPickerScreen.tsx:239]: Love this component! 😍

File: apps/mobile/src/screens/NftSelectorScreen/NftSelectorPickerGrid.tsx [hunk 2]
```diff
@@ -53,20 +53,19 @@ type NftSelectorPickerGridProps = {
     sortView: NftSelectorSortView;
   };
   screen: ScreenWithNftSelector;
-  queryRef: NftSelectorPickerGridFragment$key;
+  // queryRef: NftSelectorPickerGridFragment$key;
```
Comment (jakzaizzat) [inline apps/mobile/src/screens/NftSelectorScreen/NftSelectorPickerGrid.tsx:56]: nit: remove the comment

**Review by jakzaizzat — APPROVED** [review]
Looks good! Definitely much faster ⚡

File: apps/mobile/src/screens/NftSelectorScreen/NftSelectorPickerGrid.tsx (hunk 2 above)
Comment (kaitoo1) [reply apps/mobile/src/screens/NftSelectorScreen/NftSelectorPickerGrid.tsx:56]: oops good catch , had auto merge on lol ill fix in follow up PR

**Pull request merged**